__tests__/step-1/lottoResult.test.js

```diff
@@ -8,7 +8,11 @@ describe("Lotto Result Class Test .", () => {
   let lottoResult;
 
   beforeEach(() => {
-    lottoResult = new LottoResult(winningNumbers, bonusNumber);
+    lottoResult = new LottoResult({
+      winningNumbers: winningNumbers,
+      bonusNumber: bonusNumber,
+      tickets: ticket,
+    });
```

Comment:
오 파라미터를 객체로 받도록 변경해주셨군요 한결 보기 편하네용 ㅎㅎ key와 value가 같은 경우 아래처럼 축약해주셔도 좋을거 같습니다 !

```suggestion
    lottoResult = new LottoResult({
      winningNumbers,
      bonusNumber,
      tickets: ticket,
    });
```